server/app.js

```diff
@@ -145,6 +145,16 @@ app.get('/api/getPinMappings', (req, res) => {
 	return res.send(mappings);
 });
 
+app.get('/api/getKeyMappings', (req, res) => {
+	console.log('/api/getKeyMappings');
+	let mappings = { ...baseButtonMappings };
+	for (let prop of Object.keys(baseButtonMappings)) {
+		if (mappings[prop])
+			mappings[prop] = parseInt(baseButtonMappings[prop].key);
+	}
+	return res.send(mappings);
+});
+
```

Comment:
Heya!

If i'm not misunderstanding, checking `mappings[prop]` might not be needed as `baseButtonMappings` is defined as a const earlier so we know the structure does not contain undefined, same for `parseInt`.
 
 I see lodash is available so the entire route could be changed to:
```javascript
app.get('/api/getKeyMappings', (req, res) => 
	res.send(mapValues(baseButtonMappings, 'key')));
```

A declarative alternative without lodash:
```javascript
  const mappings = Object.entries(baseButtonMappings).reduce(
    (acc, [key, config]) => ({ ...acc, [key]: config.key }),
    {}
  );
  return res.send(mappings);
```

A Imperative alternative without lodash:
```javascript
  let mappings = {};
  for (const [key, value] of Object.entries(baseButtonMappings)) {
    mappings[key] = value.key;
  }
  return res.send(mappings);
```